lib/languages/java.js

```diff
@@ -29,7 +29,7 @@ JavaParser.prototype.parse_function = function(line) {
         // Modifiers
         '(?:(public|protected|private|static|abstract|final|transient|synchronized|native|strictfp)\\s+)*' +
         // Return value
-        '(?P<retval>[a-zA-Z_$][\\<\\>\\., a-zA-Z_$0-9]+)\\s+' +
+        '(?P<retval>[a-zA-Z_$\[\]][\\<\\>\\., a-zA-Z_$0-9\[\]]+)\\s+' +
```

Comment:
Here two backslashes are needed one to escape the square brackets in the regex and one to escape the later backslash in the string. This is probably the it breaks the build.